{% extends "layout.html" %}

{% block title %}Courses{% endblock %}

<!-- Header -->
{% block header2 %}

<br><br><br><br>
<div class="ocustom-bigheader">Courses</div>
<br>

{% endblock %}

<!-- Content -->
{% block content %}

<div style="color: black;">

<p><strong>Title:</strong></p>

<p>Quantitative and Computational Aspects of Infectious Diseases (G4016)</p>

<p><strong>Summary:</strong></p> 

<p>
The recent explosion of biological data has underscored the need of theoretical, mathematical and computational frameworks to extract fundamental knowledge in these systems. We will provide an introduction to different quantitative techniques to understand the propagation and molecular evolution of infectious agents. First, we will provide an overview of basic concepts as mutation and recombination rates as well as the theory of quasispecies. Then we will focus on the use of simple dynamical systems to model intrahost infection and epidemiological models.  We will use HIV as a case study. The third part of the course will delve into molecular evolutionary techniques, including a general introduction to population genetics, phylogenetic techniques and phylogenetic networks. We will use different infectious agents as case studies, including the influenza, HIV and dengue.
</p>

<p>
Invited lectures will cover modeling intrahost evolution and effectiveness of therapies, an introduction to genomic technologies and data analysis, an introduction to immunology and molecular epidemiology, and finally retroviruses, influenza and malaria.
</p>

<p>
Due to the large scope of this course, during the lectures each topic will be covered at an introductory level. Problems will be assigned to each lecture so students can work at home and get familiarized with the topics. 
</p>

<p><strong>Requirements:</strong></p>

<p>
This course is intended for computational biologists and biomedical informatics students interested in understanding infectious diseases as well as students from epidemiology, immunology and virology interested in learning quantitative methods for their research. Although not required, it is strongly recommended that students have a basic knowledge of biological systems, linear algebra (vector spaces and matrices), first order linear differential equations and basic programming.
</p>

<p><strong>Topics:</strong></p>
<ol>
  <li>Introduction to viruses. Quantitative understanding of viruses, mutation and evolutionary rates. Quasispecies.</li>
  <li>Mathematical modeling of Infectious Diseases: dynamical systems.</li>
  <li>Viral dynamics. </li>
  <li>Epidemiological models. </li>
  <li>Genomics. Sanger and Next Generation sequencing.</li>
  <li>Introduction to population genetics. </li>
  <li>Phylogenetics. Trees and networks.</li>
  <li>Molecular Epidemiology and Phylogeography.</li>
</ol>

<p><strong>Faculty name:</strong> Raul Rabadan</p>
<p><strong>Department:</strong> Biomedical Informatics</p>
<p><strong>Semester:</strong> Spring 2014</p>
<p><strong>Place:</strong> Room 817 Irving Cancer Research Center (Medical Center Campus)</p>
<p><strong>Schedule:</strong> M 12:30-3:30</p>
<p><strong>Grading:</strong> 
<br>
25% Problem Sets.
<br>
25% Midterm.
<br>
50% Presentation.
</p>

<p>
<strong>Reading:</strong> 
<ul>
<li>Edward Holmes, The Evolution and Emergence of RNA Viruses, Oxford Series in Ecology and Evolution, 2009.</li>
<li>Roderick Page and Edward Holmes, Molecular Evolution: A Phylogenetic Approach, Blackwell, 1998.</li>
<li>Martin Nowak, Evolutionary Dynamics: Exploring the Equations of Life, Harvard University Press, 2006.</li>
<li>Matthew Hamilton, Population Genetics, Wiley-Blackwell, 2009.</li>
<li>Jotun Hein, Mikkel H. Schierup and Carten Wiuf, Gene Genealogies, Variation and Evolution: A primer in Coalescence Theory, Oxford 2010.</li>
<li>Durbin et al., Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids, Cambridge University Press, 1998.</li>
<li>D. Huson, R. Rupp and C. Scornavacca, Phylogenetic Networks, Cambridge University, 2010.</li>
</ul>
</p>

<p><strong>Detailed Program:</strong> 

<p><u>INTRODUCTION</u>:</p>

<p>
1) Motivation to quantitative methods to infectious diseases: cases.
<ul>
<li>epidemiology: Bernuilli and small pox.</li>
<li>molecular evolution: influenza pandemias.</li>
<li>therapies and resistence: intrahost evolution of HIV, therapies.</li>
</ul>
</p>

<p>
2) Introduction to infectious diseases. Viruses, classification, structure.
</p>

<p><u>GENOMICS, SEQUENCES, POPULATION GENETICS, PHYLOGENETICS AND NETWORKS:</u></p>

<p>
3) Introduction to sequence space, mutation, recombination, reassortment, fitness, quasi species.
History: 
<ul>
<li>Eigen 1979: theory of quasi species.</li>
<li>Domingo 1980: applied to viruses.</li>
</ul>
</p>

<p>
4) Phylogenetics trees: definitions, number of trees, the space of trees, exploring the space of trees. 
</p>

<p>
5) Phylogenetics algorithms: parsimony, distance-based methods, likelihood and Bayesian.
History:
<ul>
<li>parsimony: lex parsimoniae or Occam's razor (1287 - 1347).</li>
<li>Sokal 1965: parsimony.</li>
<li>Felsestein 1981: ML.</li>
</ul>
</p>

<p>
6) Phylogenetic networks: split networks, median networks and recombinations.
History:
<ul>
<li>Buneman 1980s</li>
<li>Dress, Huson, etc. </li>
</ul>
</p>

<p>
7) Population genetics: coalescence, Wright-Fisher model.
History:
<ul>
<li>Darwin (1859) and blending problem.</li>
<li>Mendel (1900) quantized genetic information. Hardy and Weinberg solve problem independently.</li>
<li>1930s: Wright, Fisher, Haldane: founders.</li>
<li>1960s: Kimura: neutral theory of evolution, diffusion equations.</li>
<li>1982: Kingman: coalescence:</li>
<li>now: from population genetics to population genomics. </li>
</ul>
</p>

<p><u>DYNAMICAL SYSTEMS, INTRAHOST EVOLUTION AND EPIDEMIOLOGY:</u></p>

<p>
8) Introduction to dynamical systems. Simple systems. Critical points. Stability. Lodka-Volterra.
History:
<ul>
<li>1838 Verhults:logistic equation.</li>
<li>1892: Bendixson-Poincare theorem.</li>
<li>1925: Lodka-Volterra equations.</li>
</ul>
</p>

<p>
9) Viral dynamics. Modeling HIV infection.
History:
<ul>
<li>1990s: May, Nowak, Perelson.</li>
</ul>
</p>

<p>
10) Epidemiological models. Bernuilli. Verhults SI. 
History:
<ul>
<li>1760: Daniel Bernulli: model of effects of inoculation of small pox.</li>
<li>1838: Verhults SI model. </li>
<li>1927: Kermak-McKendrick SIR model.</li>
</ul>
</p>
10) Epidemiological models. Bernuilli. Verhults SI. 

</div>

{% endblock %}
