{% extends "layout.html" %}

{% block title %}Home{% endblock %}

<!-- Header -->
{% block header %}

<br> <br> <br> <br> <br>

{% endblock %}

<!-- Content -->
{% block content %}

<!-- Marketing messaging and featurettes
================================================== -->
<!-- Wrap the rest of the page in another container to center all the content. -->

<!-- Three columns of text below the carousel -->
<div class="row">
  <div class="col-lg-4">
    <a href="/publications">
    <img class="img-circle" src="static/img/frontpage/pubgraphic4.400.400.png" alt="image" width="200" height="200">
    </a>
    <h2><a href="/publications">Publications</a></h2>
    <p><i>A complete listing of publications from Dr. Rabadan and the Rabadan Lab</i></p>
  </div><!-- /.col-lg-4 -->
  <div class="col-lg-4">
    <a href="/research">
    <!--
    <img class="img-circle" src="static/img/frontpage/research.flu.400.400.png" alt="image" width="200" height="200">
    -->
    <img class="img-circle" src="static/img/frontpage/research.cnv.400.400.png" alt="image" width="200" height="200">
    </a>
    <h2><a href="/research">Research</a></h2>
    <p><i>An overview of research in the Rabadan Lab</i></p>
  </div><!-- /.col-lg-4 -->
  <div class="col-lg-4">
    <a href="/people">
    <img class="img-circle" src="static/img/frontpage/collage.300.300.png" alt="image" width="200" height="200">
    </a>
    <h2><a href="/people">People</a></h2>
    <p><i>Information about present and past members of the Rabadan Lab</i></p>
  </div><!-- /.col-lg-4 -->
</div><!-- /.row -->

<!-- START THE FEATURETTES -->

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/exotome_1000.jpg" alt="image">

  </div>

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title">Noncoding RNA</h2>

    <p class="lead">
    We have ablated the cellular RNA degradation machinery in differentiated B cells and pluripotent embryonic stem (ES) cells by conditional mutagenesis of core (Exosc3) and nuclear RNase (Exosc10) components of RNA exosome and identified a vast number of novel long non-coding RNAs (lncRNAs) and enhancer RNAs (eRNAs)
    </p>

    <small>
    <strong>RNA-Exosome-Regulated Long Non-coding RNA Transcription Controls Super-Enhancer Activity</strong> 
    <br>
    Evangelos Pefanis*, Jiguang Wang*, Gerson Rothschild*, Junghyun Lim*, et al.
    <br>
    <a href="http://www.cell.com/cell/abstract/S0092-8674(15)00490-0"><i><b>Cell</b></i> 2015 May; doi: 10.1016/j.cell.2015.04.034</a><br>
    <i>*These authors have contributed equally to this work.</i>
    </small>

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <h2 class="bs-docs-featurette-title">non-Hodgkin’s lymphoma</h2>

    <p class="lead">
    <i>"The first-ever systematic study of the genomes of patients with ALK-negative anaplastic large cell lymphoma (ALCL), a particularly aggressive form of non-Hodgkin’s lymphoma (NHL), shows that many cases of the disease are driven by alterations in the JAK/STAT3 cell signaling pathway. The study also demonstrates, in mice implanted with human-derived ALCL tumors, that the disease can be inhibited by compounds that target this pathway, raising hopes that more effective treatments might soon be developed."</i>
    <br>
    ---<a href="http://newsroom.cumc.columbia.edu/blog/2015/04/14/lymphoma-genome-study/">CUMC Newsroom</a>
    </p>

    <small>
    <b>Convergent Mutations and Kinase Fusions Lead to Oncogenic STAT3 Activation in Anaplastic Large Cell Lymphoma.</b>
    <br>
    Ramona Crescenzo*, Francesco Abate*, Elena Lasorsa*, et al
    <br>
    <a href="http://www.cell.com/cancer-cell/abstract/S1535-6108(15)00094-X"><i><b>Cancer Cell</b></i> 2015 April 13; doi:10.1016/j.ccell.2015.03.006.</a><br>
    <i>* These authors have contributed equally to this work.</i>
    </small>

  </div>

  <div class="col-md-7">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/rabadan_ani_fade.gif" alt="image">

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/exotomelarge.png" alt="image">

  </div>

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title">Noncoding RNA</h2>

    <p class="lead">
    The transcriptome of Exosc3-deficient B cells has revealed the presence of many novel RNA exosome substrate ncRNAs. xTSS-RNAs are most strongly expressed at genes that accumulate AID-mediated somatic mutations and/or are frequent translocation partners of DNA double-strand breaks generated at Igh in B cells. Strikingly, translocations near TSSs or within gene bodies occur over regions of RNA exosome substrate ncRNA expression. 
    </p>

    <br>
    <br>

    <small>
    <strong>Noncoding RNA transcription targets AID to divergently transcribed loci in B cells</strong> 
    <br>
    Evangelos Pefanis*, Jiguang Wang*, Gerson Rothschild*, Junghyun Lim, Jaime Chao, Raul Rabadan<sup>#</sup>, Aris N. Economides, Uttiya Basu<sup>#</sup>. 
    <br>
    <a href="http://www.nature.com/nature/journal/v514/n7522/abs/nature13580.html" target="_blank"><i><b>Nature</b></i> 2014 Oct 16;514(7522):389-93. doi: 10.1038/nature13580.</a>
    <br>
    <i>*These authors have contributed equally to this work. <sup>#</sup>Corresponding authors.</i>
    </small>

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-7">
    <h2 class="bs-docs-featurette-title">Chronic Lymphocytic Leukemia</h2>
    <p class="lead">
    A graph representing the sequence of genomic alterations in chronic lymphocytic leukemia (CLL). Each node represents a mutation, with arrows indicating temporal relationships between them. The size of the nodes indicates the number of patients in the study who exhibited the alteration, while the thickness of the lines shows how often the temporal relationships between nodes were seen. The method the researchers use enabled them to identify multiple, distinct evolutionary patterns in CLL.
    </p>

    <small>
    <strong>Tumor evolutionary directed graphs and the history of chronic lymphocytic leukemia.</strong><br>
    Jiguang Wang*, Hossein Khiabanian*, Davide Rossi*, et al.<br />
    <a href="http://elifesciences.org/content/3/e02869" target="_blank"><i><b>eLife</b></i> 2014 Dec 11; doi: 10.7554/eLife.02869.</a><br>
    <i>* These authors have contributed equally to this work.</i>
    </small>

  </div>

  <div class="col-md-5">
    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/map_1000.jpg" alt="image">
  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/cancerfusion_1000.jpg" alt="image">

  </div>

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title">Glioblastoma</h2>

    <p class="lead">
    <i>"Graphic representation of the collaboration between experimental and computational biology. The outer ring represents results of next-generation genetic sequencing of the glioblastoma genome, showing expression of the FGFR-TACC fusion gene (red peaks). In the center, FGFR-TACC fusion protein (red) can be seen disrupting tubulin bundles (green), structures that support cell division, or mitosis, at the point connecting the two daughter cells (whose nuclei are colored blue)."</i>
    <br>
    ---<a href="http://newsroom.cumc.columbia.edu/blog/2012/07/26/research-may-lead-to-new-treatment-for-type-of-brain-cancer/">CUMC Newsroom</a>
    </p>

    <br>
    <br>
    <br>
    <br>

    <small>
    <b>Transforming Fusions of FGFR and TACC Genes in Human Glioblastoma.</b>
    <br>
    Devendra Singh*, Joseph Minhow Chan*, Pietro Zoppoli*, Francesco Niola*, et al.
    <br>
    <a href="http://www.sciencemag.org/content/early/2012/07/25/science.1220834" target="_blank"><i><b>Science</b></i> 2012 10.1126/science.1220834.</a><br>
    <i>* These authors have contributed equally to this work.</i>
    </small>

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title">Tumor Evolution</h2>
    <p class="lead">
    Tumor evolutionary modes visualized in PΣ3. <i>A</i>: frozen evolution, <i>B</i>: branched evolution, <i>C</i>: divergent evolution, <i>D</i>: linear evolution, <i>E</i>: somatic hypermutation.
    </p>
  </div>

  <div class="col-md-5">
    <img class="featurette-image img-responsive center-block" src="static/img/frontpage/fat-tri_1000.jpg" alt="image">
  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">
    <img class="featurette-image img-responsive center-block" src="static/img/frontpage/mecaenrich_1000.png" alt="image">
  </div>

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title">Title</h2>
    <p class="lead">
    text
    </p>
  </div>

</div>

{% endblock %}
