{% extends "layout.html" %}

{% block title %}Research{% endblock %}

<!-- Header -->
{% block header2 %}

<br><br><br><br>
<div class="ocustom-bigheader">Research Highlights</div>
<br>

{% endblock %}

<!-- Content -->
{% block content %}

<!-- START THE FEATURETTES -->

<div class="row featurette">

  <div class="col-md-5">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/exotome_1000.jpg" alt="image">

  </div>

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">Identifying Novel Noncoding RNAs</h2>

    <p class="lead">
    We have ablated the cellular RNA degradation machinery in differentiated B cells and pluripotent embryonic stem (ES) cells by conditional mutagenesis of core (Exosc3) and nuclear RNase (Exosc10) components of RNA exosome and identified a vast number of novel long non-coding RNAs (lncRNAs) and enhancer RNAs (eRNAs)
    </p>

    <small>
    <strong class="ocustom-bitter">RNA-Exosome-Regulated Long Non-coding RNA Transcription Controls Super-Enhancer Activity</strong> 
    <br>
    Evangelos Pefanis*, Jiguang Wang*, Gerson Rothschild*, Junghyun Lim*, et al.
    <br>
    <a href="http://www.cell.com/cell/abstract/S0092-8674(15)00490-0"><i><b>Cell</b></i> 2015 May; doi: 10.1016/j.cell.2015.04.034</a><br>
    <i>*These authors have contributed equally to this work.</i>
    </small>

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">Connections between Mendelian Diseases and Cancer</h2>

    <p class="lead">
    <!--
    <i>"Comorbidity between Mendelian disease and cancer may be due to germline mutations that provide a fertile ground for the growth of certain aberrant cells.”</i>
    -->
    <i>"If germline genetic variation in Mendelian loci predisposes bearers to common cancers, the same loci may harbour cancer-associated somatic variation. Compilations of clinical records spanning over 100 million patients provide an unprecedented opportunity to assess clinical associations between Mendelian diseases and cancers. We systematically compare these comorbidities against recurrent somatic mutations from more than 5,000 patients across many cancers. Using multiple measures of genetic similarity, we show that a Mendelian disease and comorbid cancer indeed have genetic alterations of significant functional similarity."</i>
    <br>
    ---<a href="http://www.nature.com/ncomms/2015/150430/ncomms8033/full/ncomms8033.html">Nature Communications</a>
    </p>

    <small>
    <strong class="ocustom-bitter">Genetic similarity between cancers and comorbid Mendelian diseases identifies candidate driver genes.</strong> 
    <br>
    Rachel Melamed, Kevin Emmett, Chioma Madubata, et al.
    <br>
    <a href="http://www.nature.com/ncomms/2015/150430/ncomms8033/full/ncomms8033.html"><i><b>Nature Communications</b></i> 2015 April 30; doi:10.1038/ncomms8033.</a>
    </small>

  </div>

  <div class="col-md-7">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/rachel.cormob.310.jpg" alt="image">

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-7">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/rabadan_ani_fade.gif" alt="image">

  </div>

  <div class="col-md-5">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">Non-Hodgkin’s Lymphoma</h2>

    <p class="lead">
    <i>"The first-ever systematic study of the genomes of patients with ALK-negative anaplastic large cell lymphoma (ALCL), a particularly aggressive form of non-Hodgkin’s lymphoma (NHL), shows that many cases of the disease are driven by alterations in the JAK/STAT3 cell signaling pathway. The study also demonstrates, in mice implanted with human-derived ALCL tumors, that the disease can be inhibited by compounds that target this pathway, raising hopes that more effective treatments might soon be developed."</i>
    <br>
    ---<a href="http://newsroom.cumc.columbia.edu/blog/2015/04/14/lymphoma-genome-study/">CUMC Newsroom</a>
    </p>

    <small>
    <strong class="ocustom-bitter">Convergent Mutations and Kinase Fusions Lead to Oncogenic STAT3 Activation in Anaplastic Large Cell Lymphoma.</strong> 
    <br>
    Ramona Crescenzo*, Francesco Abate*, Elena Lasorsa*, et al
    <br>
    <a href="http://www.cell.com/cancer-cell/abstract/S1535-6108(15)00094-X"><i><b>Cancer Cell</b></i> 2015 April 13; doi:10.1016/j.ccell.2015.03.006.</a><br>
    <i>* These authors have contributed equally to this work.</i>
    </small>

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">Chronic Lymphocytic Leukemia</h2>

    <p class="lead">
    A graph representing the sequence of genomic alterations in chronic lymphocytic leukemia (CLL). Each node represents a mutation, with arrows indicating temporal relationships between them. The size of the nodes indicates the number of patients in the study who exhibited the alteration, while the thickness of the lines shows how often the temporal relationships between nodes were seen. The method the researchers use enabled them to identify multiple, distinct evolutionary patterns in CLL.
    </p>

    <small>
    <strong class="ocustom-bitter">Tumor evolutionary directed graphs and the history of chronic lymphocytic leukemia.</strong>
    <br>
    Jiguang Wang*, Hossein Khiabanian*, Davide Rossi*, et al.
    <br>
    <a href="http://elifesciences.org/content/3/e02869" target="_blank"><i><b>eLife</b></i> 2014 Dec 11; doi: 10.7554/eLife.02869.</a><br>
    <i>* These authors have contributed equally to this work.</i>
    </small>

  </div>

  <div class="col-md-7">
    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/map_1000.jpg" alt="image">
  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-7">

    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/exotomelarge.png" alt="image">

  </div>

  <div class="col-md-5">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">Identifying Novel Noncoding RNAs</h2>

    <p class="lead">
    The transcriptome of Exosc3-deficient B cells has revealed the presence of many novel RNA exosome substrate ncRNAs. xTSS-RNAs are most strongly expressed at genes that accumulate AID-mediated somatic mutations and/or are frequent translocation partners of DNA double-strand breaks generated at Igh in B cells. Strikingly, translocations near TSSs or within gene bodies occur over regions of RNA exosome substrate ncRNA expression. 
    </p>

    <br>
    <br>

    <small>
    <strong class="ocustom-bitter">Noncoding RNA transcription targets AID to divergently transcribed loci in B cells</strong> 
    <br>
    Evangelos Pefanis*, Jiguang Wang*, Gerson Rothschild*, Junghyun Lim, Jaime Chao, Raul Rabadan<sup>#</sup>, Aris N. Economides, Uttiya Basu<sup>#</sup>. 
    <br>
    <a href="http://www.nature.com/nature/journal/v514/n7522/abs/nature13580.html" target="_blank"><i><b>Nature</b></i> 2014 Oct 16;514(7522):389-93. doi: 10.1038/nature13580.</a>
    <br>
    <i>*These authors have contributed equally to this work. <sup>#</sup>Corresponding authors.</i>
    </small>

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">Bacterial Evolution</h2>

    <p class="lead">
    Topological network representation of <i>S. aureus</i> genome profiles. Color corresponds to enrichment in mecA, an antibiotic resistance gene.
    </p>

    <br>
    <small>
    <strong class="ocustom-bitter">Characterizing Scales of Genetic Recombination and Antibiotic Resistance in Pathogenic Bacteria Using Topological Data Analysis.</strong>
    <br>
    Kevin Emmett and Raul Rabadan.
    <br>
    <i><b>Lecture Notes in Computer Science</b></i> 2014. Volume 8609, pp 540-551.
    <br>
    <a href="http://arxiv.org/abs/1406.1219">arXiv:1406.1219.</a>
    </small>

  </div>

  <div class="col-md-7">
    <img class="featurette-image img-responsive center-block" src="static/img/frontpage/mecaenrich_1000.png" alt="image">
  </div>

</div>

<!--
<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-5">

    <br>
    <br>

    <img class="featurette-image img-responsive center-block" src="/static/img/frontpage/cancerfusion_1000.jpg" alt="image">

  </div>

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title">Glioblastoma</h2>

    <p class="lead">
    <i>"Graphic representation of the collaboration between experimental and computational biology. The outer ring represents results of next-generation genetic sequencing of the glioblastoma genome, showing expression of the FGFR-TACC fusion gene (red peaks). In the center, FGFR-TACC fusion protein (red) can be seen disrupting tubulin bundles (green), structures that support cell division, or mitosis, at the point connecting the two daughter cells (whose nuclei are colored blue)."</i>
    <br>
    ---<a href="http://newsroom.cumc.columbia.edu/blog/2012/07/26/research-may-lead-to-new-treatment-for-type-of-brain-cancer/">CUMC Newsroom</a>
    </p>

    <br>
    <br>

    <small>
    <b>Transforming Fusions of FGFR and TACC Genes in Human Glioblastoma.</b>
    <br>
    Devendra Singh*, Joseph Minhow Chan*, Pietro Zoppoli*, Francesco Niola*, et al.
    <br>
    <a href="http://www.sciencemag.org/content/early/2012/07/25/science.1220834" target="_blank"><i><b>Science</b></i> 2012 10.1126/science.1220834.</a><br>
    <i>* These authors have contributed equally to this work.</i>
    </small>

  </div>

</div>
-->

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-7">

    <img class="featurette-image img-responsive center-block" src="static/img/frontpage/fat-tri_1000.jpg" alt="image">

  </div>

  <div class="col-md-5">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">Tumor Evolution</h2>

    <p class="lead">
    Tumor evolutionary modes visualized in PΣ3. 
    <li><i>A</i>: frozen evolution</li>
    <li><i>B</i>: branched evolution</li>
    <li><i>C</i>: divergent evolution</li>
    <li><i>D</i>: linear evolution</li>
    <li><i>E</i>: somatic hypermutation</li>
    </p>

    <br>
    <br>

    <small>
    <strong class="ocustom-bitter">Moduli Spaces of Phylogenetic Trees Describing Tumor Evolutionary Patterns.</strong>
    <br>
    Sakellarios Zairis, Hossein Khiabanian, Andrew Blumberg, and Raul Rabadan.
    <br>
    <a href="http://link.springer.com/chapter/10.1007/978-3-319-09891-3_48" target="_blank"><i><b>Lecture Notes in Computer Science</b></i> 2014. Volume 8609, pp 528-539. doi: 10.1007/978-3-319-09891-3_48.</a><br>
    <a href="http://arxiv.org/abs/1410.0980" target="_blank">arXiv:1410.0980 [full version].</a>
    </small>

  </div>

</div>

<hr class="featurette-divider">

<div class="row featurette">

  <div class="col-md-7">

    <h2 class="bs-docs-featurette-title ocustom-droidsans">A Topological Approach to Modeling Evolution</h2>

    <p class="lead">
    <i>"Recent genomic studies have made it clear that evolution does not only proceed in a 'vertical' pattern in which one organism inherits genomic information from the organisms from which it descends (figure A). Scientists now understand that genomic evolution can also be 'horizontal'; that is, genomic information can be transferred between organisms or evolutionarily similar groups of organisms that are in parallel lineages (figure B), such as in cases of species hybridization in eukaryotes, lateral gene transfer in bacteria, recombination and reassortment in viruses, viral integration in eukaryotes, and fusion of genomes of symbiotic species. These observations suggest that phylogenetic trees have limitations in their ability to characterize evolution at the molecular level and that another model is needed that can integrate both vertical and horizontal evolution.</i>
    <br>
    ---<a href="http://systemsbiology.columbia.edu/news/a-topological-approach-to-modeling-evolution">CU Systems Biology News</a>
    </p>

    <small>
    <strong class="ocustom-bitter">Topology of viral evolution.</strong>
    <br>
    Chan JM, Carlsson G, Rabadan R.
    <br>
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/24170857" target="_blank"><i><b>Proc Natl Acad Sci USA</b></i> 2013 Oct 29. doi: 10.1073/pnas.1313480110.</a>
    </small>

  </div>

  <div class="col-md-5">
    <img style="padding:45px 0px 0px 0px;" class="featurette-image img-responsive center-block" src="static/img/frontpage/pnas.1313480110fig01.jpg" alt="image">
  </div>

</div>


{% endblock %}
